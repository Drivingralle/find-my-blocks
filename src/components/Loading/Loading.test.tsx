import React from "react";
import renderer from "react-test-renderer";
import { Loading } from "./";

describe("Loading component should", () => {
  it("render correctly", () => {
    const tree = renderer.create(<Loading>Loading</Loading>).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
