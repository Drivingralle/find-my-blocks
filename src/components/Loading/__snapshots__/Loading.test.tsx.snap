// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="loading"
>
  <div
    className="square"
  />
  <div
    className="square"
  />
  <div
    className="square"
  />
  <div
    className="square"
  />
  <div
    className="square"
  />
  <div
    className="square"
  />
  <div
    className="square"
  />
</div>
`;
